### DISCUSSION TITLE
Basic Git Commands on LinkedIn

### DISCUSSION THREAD

**Discussion by jason-muljadi** [albertkun/21S-ASIAAM-191A#51] [General]
I saw this post and I hope it helps folks on the basics of Git!

Credit: Tommy Tho / https://www.linkedin.com/posts/tommythowm_git-figma-programming-activity-6780718722488385536-iC3y

![image](https://user-images.githubusercontent.com/81675803/113742378-6c97c780-96b7-11eb-950b-51bb9a7c8a91.png)

![image](https://user-images.githubusercontent.com/81675803/113742403-73263f00-96b7-11eb-846b-417de44c7f8c.png)

![image](https://user-images.githubusercontent.com/81675803/113742455-80dbc480-96b7-11eb-93c5-ac301e87b7dc.png)

![image](https://user-images.githubusercontent.com/81675803/113742502-8933ff80-96b7-11eb-9b4c-5c9571a50221.png)

![image](https://user-images.githubusercontent.com/81675803/113742519-8cc78680-96b7-11eb-91d5-8cc0c88f0c8a.png)

![image](https://user-images.githubusercontent.com/81675803/113742544-918c3a80-96b7-11eb-98d8-f612701959a0.png)

![image](https://user-images.githubusercontent.com/81675803/113742579-981ab200-96b7-11eb-87d7-57348a798ee5.png)

![image](https://user-images.githubusercontent.com/81675803/113742597-9d77fc80-96b7-11eb-980b-8c0f3607d48a.png)

**Comment by alizezila** [comment]
thank you for posting! I've never used git before so this guide is very helpful!
